lib/dfe/reference_data/itt/subjects.rb

```diff
@@ -743,7 +786,8 @@ module ITT
                                                       publish_category: 'W1',
                                                       incentive: {
                                                         '2020-2021' => 'd4024ca2-da1a-47ff-b497-62fed8fa5249',
-                                                        '2024-2025' => 'a324ae47-d9db-4eb5-9039-c767b8ec4f08'
+                                                        '2024-2025' => 'a324ae47-d9db-4eb5-9039-c767b8ec4f08',
+                                                        '2025-2026' => '4427e0e7-e771-4538-8224-be8d570c7996'
```

Comment:
Both 2025 and 2026 link to art and design, but shouldn't graphic design be linked to D&T? Who can we check with?

Reply:
I've removed the link to the incentive for 2025-2026 and created --> https://trello.com/c/WEwqapWg/2312-dfe-reference-data-confirm-if-graphic-design-should-be-associated-with-a-dt-incentive-or-an-ad-incentive?filter=member:portererica